spec/spec.yaml

```diff
@@ -234,8 +234,15 @@ paths:
             "installation_name": "shire",
             "provider": "aws",
             "datacenter": "eu-central-1",
-            "supported_k8s_version_skews": [
-              "1.17", "1.18", "1.19"
+            "kubernetes_versions": [
+              {
+                "minor_version": "1.15",
+                "eol_date": "2020-03-24T00:00:47.215Z"
+              },
+              {
+                "minor_version": "1.16",
+                "eol_date": "2020-08-26T00:00:47.215Z"
+              }
```

Comment:
I would prefer to not have the time part included (`T00:00:47.215Z`). We don't have that information, and a date is accurate enough.

Reply:
I changed the format to `date`, so we won't have to include the time 👍🏻